tsconfig.json

```diff
@@ -3,28 +3,28 @@
   "files": [],
   "references": [
     {
-      "path": "./.github/workflows/scripts"
+      "path": "./.github/workflows/scripts/tsconfig.json"
```

Comment:
Did this work before? Absolutely good to be specific here, just curious if there's any other reason for doing so!

Reply:
It did, but a recent Visual Studio update broke CMD + click on the paths 🤷‍♂️